src/utils/createEmbedding.ts

```diff
@@ -0,0 +1,54 @@
+import dotenv from 'dotenv';
+import * as path from 'path';
+import { Configuration, OpenAIApi, CreateEmbeddingRequest } from 'openai';
+dotenv.config({ path: path.resolve(__dirname, '../..', '.env.local') });
```

Comment:
is this not causing a linting error if it's before the final import?

Reply:
No, but changed anyway